src/components/ClubHome/Activities/Activities.js

```diff
@@ -1,91 +1,121 @@
-    date: "2021-05-31",
-    desc: "나",
-  },
-  {
-    title: "오창훈",
-    img: "https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2Fe5b6456c-10ed-4570-b60d-f3a5c4c729f6%2F.jpeg?table=block&id=e2706cde-8e3d-408a-b6ad-326c296a7f05&spaceId=065109dd-4d22-4e3f-9455-ff3805b7d264&width=2940&userId=601054f0-a611-49e0-84b9-04831ec920f4&cache=v2",
-    date: "2021-06-31",
-    desc: "넵!",
-  },
-  {
-    title: "이석호",
-    img: "https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F08e6b24f-580c-4158-9b54-b943eb522858%2F.jpg?table=block&id=a1bffbb2-7e08-4e04-9e50-b395b426e9b9&spaceId=065109dd-4d22-4e3f-9455-ff3805b7d264&width=2160&userId=601054f0-a611-49e0-84b9-04831ec920f4&cache=v2",
-    date: "2021-07-31",
-    desc: "형 담배 그렇게 피면 죽어요...",
-  },
-  {
-    title: "김지수",
-    img: "https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2Fa7a90cb9-6987-46dc-b5a1-8d27ab68bfda%2F.jpeg?table=block&id=d0c34eb2-1dee-486f-b1bc-236fa6c83732&spaceId=065109dd-4d22-4e3f-9455-ff3805b7d264&width=2700&userId=601054f0-a611-49e0-84b9-04831ec920f4&cache=v2",
-    date: "2021-08-31",
-    desc: "영양제 귀신",
-  },
-  {
-    title: "류가희",
-    img: "https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F0418e7e1-30bc-4519-b893-173977071a0a%2F.jpeg?table=block&id=fd4c1f3a-40f8-4358-bcd6-7fac02f3199d&spaceId=065109dd-4d22-4e3f-9455-ff3805b7d264&width=1500&userId=601054f0-a611-49e0-84b9-04831ec920f4&cache=v2",
-    date: "2021-09-31",
-    desc: "인천 통학러",
-  },
+    "no": 160,
+    "title": "제목수정",
+    "studentId": "test1",
+    "studentName": "test1",
+    "clubName": "우아한 애자일",
+    "category": "IT",
+    "inDate": "2021-09-16 11:16:32",
+    "modifyDate": "2021-09-27 11:31:42",
+    "url": null,
+    "fileId": null,
+    "hit": 137
+  }
 ];
 
-const Activities = ({ onModalOpen }) => {
-  const onCreateActivies = () => {
-    console.log("클릭됨");
+const Activities = () => {
+  const [isModalOpened, setIsModalOpened] = useState(false);
+  const [selectedID, setSelectedID] = useState();
+  const router = useRouter();
+
+  const onClick = (id) => {
+    setSelectedID(id)
+    setIsModalOpened(true);
   };
+  const onClose = () => {
+    setSelectedID();
+    setIsModalOpened(false);
+  }
 
   return (
     <div className={styles.container}>
       <div id={styles.clubName}>
         <p>우아한 애자일의 활동</p>
       </div>
       <div id={styles.add}>
-        <IoIosAddCircleOutline onClick={onCreateActivies} />
+        <Link href={`${router.pathname}/writeActivity`} passHref><IoIosAddCircleOutline /></Link>
       </div>
       <div className={styles.activities}>
         {actData.map((el, i) => {
```

Comment:
```suggestion
        {actData.map((act, index) => {
```
물론 내가 해놓은거긴 한데 이렇게 바꿔주는게 좋대요 ㅋㅋㅋ